ui/components/Page.tsx

```diff
@@ -0,0 +1,184 @@
+  padding-top: ${(props) => props.theme.spacing.medium};
+  padding-right: ${(props) => props.theme.spacing.medium};
+  background-color: ${negativeSpaceColor};
+`;
+
+const NavContent = styled.div`
+  background-color: white;
+  height: 100vh;
+  padding-top: ${(props) => props.theme.spacing.medium};
+  padding-left: ${(props) => props.theme.spacing.xs};
+  padding-right: ${(props) => props.theme.spacing.xl};
+
+  .MuiTab-wrapper {
+    text-transform: capitalize;
+    font-size: 20px;
+    font-weight: bold;
+  }
+
+  .MuiTabs-indicator {
+    left: 0;
+    width: 4px;
+    background-color: ${(props) => props.theme.colors.primary};
+  }
+`;
+
+const ContentContainer = styled.div`
+  width: 100%;
+  padding-top: ${(props) => props.theme.spacing.medium};
+  padding-bottom: ${(props) => props.theme.spacing.medium};
+  padding-right: ${(props) => props.theme.spacing.medium};
+`;
+
+const Content = styled.div`
+  max-width: 1400px;
+  margin: 0 auto;
+  background-color: ${(props) => props.theme.colors.white};
+  padding-left: ${(props) => props.theme.spacing.large};
+  padding-right: ${(props) => props.theme.spacing.large};
+  padding-top: ${(props) => props.theme.spacing.large};
+  padding-bottom: ${(props) => props.theme.spacing.medium};
+`;
+
+const TitleBar = styled.div`
+  margin-bottom: ${(props) => props.theme.spacing.large};
+
+  h2 {
+    margin: 0 !important;
+  }
+`;
+
+const Main = styled(Flex)`
+  flex: 1 1 auto;
+`;
+
+const TopToolBar = styled(Flex)`
+  padding: 8px 0;
+  background-color: ${(props) => props.theme.colors.primary};
+  width: 100%;
+  height: 112px;
+  flex: 0 1 auto;
+
+  ${Logo} img {
+    width: 136px;
+    height: 42px;
+  }
+`;
+
+const UserAvatar = styled(Icon)`
+  padding-right: ${(props) => props.theme.spacing.medium};
+`;
+
+function pageLookup(p: PageRoute) {
+  switch (p) {
+    case PageRoute.Applications:
+      return "Applications";
+
+    default:
+      break;
+  }
+}
+
+function Page({ className, children, title, breadcrumbs }: Props) {
+  const { currentPage } = useNavigation();
+  return (
+    <div className={className}>
+      <AppContainer>
+        <TopToolBar between align>
+          <Logo />
+          <UserAvatar size="xl" type={IconType.Account} color="white" />
+        </TopToolBar>
+        <Main wide>
+          <NavContainer>
+            <NavContent>
+              <Tabs
+                centered={false}
+                orientation="vertical"
+                value={getNavValue(currentPage)}
+              >
+                {_.map(navItems, (n) => (
+                  <StyleLinkTab
+                    value={n.value}
+                    key={n.value}
+                    label={n.label}
+                    to={formatURL(n.value)}
+                  />
+                ))}
+              </Tabs>
+            </NavContent>
+          </NavContainer>
+          <ContentContainer>
+            <Content>
+              <TitleBar>
+                <Breadcrumbs>
```

Comment:
Interesting, I like the look and feel of having this in the Content section! I think @foot suggested something similar when we were having a look at Figma. I had to come up with a workaround to have it up in the appbar as I don't have access to that sort of information at that point (the appbar is outside the Switch). So I am "pretending" it's the app bar but in fact it's part of the main page (I divided the right side of the Drawer in the section header - containing the breadcrumbs - and the page content).